File: tests/integration/deleted_with/python/fails_on_delete.py
# Copyright 2016-2023, Pulumi Corporation.  All rights reserved.

from typing import Optional

import pulumi

class FailsOnDelete(pulumi.CustomResource):
    def __init__(self,
                 resource_name: str,
                 opts: Optional[pulumi.ResourceOptions] = None):
        super().__init__("testprovider:index:FailsOnDelete", resource_name, {}, opts)

File: tests/integration/deleted_with/nodejs/failsOnDelete.ts
// Copyright 2016-2023, Pulumi Corporation.  All rights reserved.

import * as pulumi from "@pulumi/pulumi";

export class FailsOnDelete extends pulumi.CustomResource {
    constructor(name: string, opts?: pulumi.CustomResourceOptions) {
        super("testprovider:index:FailsOnDelete", name, {}, opts);
    }
}
